chunky/src/java/se/llbit/chunky/ui/render/AdvancedTab.java

```diff
@@ -105,6 +106,20 @@ public AdvancedTab() throws IOException {
       PersistentSettings.setNumRenderThreads(value);
       renderControls.showPopup("This change takes effect after restarting Chunky.", renderThreads);
     });
+
+    octreeImplementation.getItems().addAll("NODE", "PACKED", "BIGPACKED");
```

Comment:
These have to be dynamic now, something like `Octree.getImplementations()` which returns the keys of the hashmap?

Reply:
Description is created dynamically too